stdlib/public/SDK/XCTest/XCTestCaseAdditions.mm

```diff
@@ -152,12 +150,11 @@ - (NSString *)className
                    @"reason": exception.reason,
                    };
     }
-
     @catch (...) {
         result = @{
                    @"type": @"unknown",
                    };
     }
 
-    return [result retain];
+    return result;
```

Comment:
Yeah, that's what I thought based on the comment that was also removed [here](https://github.com/apple/swift/commit/222811a27ad19333b3641bdf81d1f5b7fd2c368d#diff-89e67e0e81a2188e97cbef478775749a)